Network: Follow-up on internal OVN load balancers and forwards (#16179)
canonical/lxd: doc/howto/network_forwards.md

- Forward traffic from different port numbers of the external address to different instances (and optionally different ports on those instances).
  This method allows to "share" your external IP address and expose more than one instance at a time.

> In case of the {ref}`network-ovn`, network forwards also allow an internal IP address (or specific ports on it) to be forwarded to another internal IP address (or specific ports on it).

Reviewer: ```suggestion
For {ref}`OVN networks <network-ovn>`, network forwards also allow an internal IP address (or specific ports on it) to be forwarded to another internal IP address (or specific ports).
```
